### ISSUE TITLE
[Feature Request] obj.tags.has("tagname")

### ISSUE THREAD

**Issue by kovitikus** [evennia/evennia#2204]
#### Is your feature request related to a problem? Please describe.
When using tags, sometimes it's useful to check a single object for a specific tag. This can already be accomplished with an if statement on a `get()` method call, but it's not obvious. Even if the return results are the same, True or False, it doesn't feel clean to use get for this purpose.

#### Describe the solution you'd like
I'd like to see some helper function to check an object for a tag using `has`, as it's the most common way to check something for a property. All I can think of at the moment is a string passed to the method, but maybe there's other arguments or uses that someone else may consider.

```py
if obj.tags.has("container"):
    peach.move_to(obj, quiet=True)
```

**Comment by Griatch** [comment]
Should be easy enough to add; it'd make tags consistent with the AttributeHandler api too. The callsign would likely need to be `.has(tagname, category=None)`.

**Comment by Griatch** [comment]
Closed with #2210
